Remove Unicode non-printing space, fixes XML appstream breakage.

cat utilities/texmaker.appdata.xml | tr -d $(/usr/bin/printf "\u200b") > utilities/texmaker.appdata.xml.b
mv utilities/texmaker.appdata.xml.b utilities/texmaker.appdata.xml
Index: texmaker-5.1.1/utilities/texmaker.appdata.xml
===================================================================
--- texmaker-5.1.1.orig/utilities/texmaker.appdata.xml
+++ texmaker-5.1.1/utilities/texmaker.appdata.xml
@@ -4,7 +4,7 @@
  <metadata_license>CC0-1.0</metadata_license>
  <project_license>GPL-2.0+</project_license>
  <name>Texmaker</name>
-​  <summary>Latex editor</summary>
+  <summary>Latex editor</summary>
  <description>
     <p>
     Texmaker is a clean, powerful and highly configurable LaTeX editor.
@@ -34,19 +34,19 @@
 <url type="help">https://www.xm1math.net/texmaker/doc.html</url>
 <provides>
   <binary>texmaker</binary>
-​</provides>
+</provides>
 <launchable type="desktop-id">texmaker.desktop</launchable>
  <project_group>Qt</project_group>
  <developer_name>Pascal Brachet</developer_name>
  <releases>
-​   <release version="5.1.1" date="2021-08-19">
-​     <description>
-​       <p>Maintenance release</p>
+   <release version="5.1.1" date="2021-08-19">
+     <description>
+       <p>Maintenance release</p>
      </description>
    </release>
-​   <release version="5.1.0" date="2021-08-14">
-​     <description>
-​       <p>Maintenance release - Qt6 support</p>
+   <release version="5.1.0" date="2021-08-14">
+     <description>
+       <p>Maintenance release - Qt6 support</p>
      </description>
    </release>
  </releases>
